README.md

```diff
@@ -335,6 +335,38 @@ or specify which container you want to view the logs from with
     docker logs [container-name]
 ```
 
+### Using custom TLS certificates within the container
+
+You must ensure that your custom certificate file exists inside the container
+where the transport service is running.
+
+You can achieve this by copying the certificate within a custom image or
+by mounting the file using the composition file.
+
+To create a custom image, inherit our image as a base and copy your
+file to the container path you wish, */etc/my-tls-file*, eg,
+
+```shell
+    FROM wirepas/gateway
+    COPY my-tls-file /etc/my-tls-file
+```
+
+Build the image and update the composition file to match the build name that
+you pick or use.
+
+In case you opt to mount the file within the transport service container,
```

Comment:
I would just present this second option. More option means more questions ;-) 
This second option seems enough, no?

Reply:
Agree, it is enough :+1: